.gitlab/nersc-ci-pip.yml

```diff
@@ -0,0 +1,100 @@
+variables:
+  PIP_WORK_DIR: ${SDK_WORK_DIR}/pip
+  PIP_REQUIREMENTS: .gitlab/pip-requirements.txt
+
+
+stages:
+  - setup
+  - build
+  - test
+  - cleanup
+
+
+### MACHINES
+.on_perlmutter:
+  variables:
+    SITE: "perlmutter"
+    VENV_ENV_NAME: "perlmutter-env"
+# python/3.10 works too, python/3.11 breaks on 'pip install --no-cache-dir -r ${PIP_REQUIREMENTS}' in .pip_build; fails to satisfy requirement smartsim==0.5.1
+    PYTHON_MODULE: "python/3.9"
+
+
+### SCRIPTS
+.final_steps:
+  script: &finalize
+    - chgrp -fR m3973 ${PIP_WORK_DIR} || true
+    - chmod -fR 02770 ${PIP_WORK_DIR}    || true
+
+.job_tags:
+  tags:
+    - $SITE
+    - $RUNNER_TYPE
+
+.pip_env_setup:
+  stage: setup
+  script:
+    - mkdir -p ${PIP_WORK_DIR}
+    - test -d ${PIP_WORK_DIR}/${VENV_ENV_NAME} && exit 0
+    - module add ${PYTHON_MODULE}
+    - python3 -m venv ${PIP_WORK_DIR}/${VENV_ENV_NAME}
+    - source ${PIP_WORK_DIR}/${VENV_ENV_NAME}/bin/activate
+    - pip install -U pip setuptools wheel
+    - pip cache purge
+    - *finalize
+
+.pip_build:
+  stage: build
+  script:
+    - source ${PIP_WORK_DIR}/${VENV_ENV_NAME}/bin/activate
+    - pip install --no-cache-dir -r ${PIP_REQUIREMENTS}
+    - (smart clean && smart build) > /dev/null 2>&1 || true
+    - *finalize
+
+.pip_test:
+  stage: test
+  variables:
+    MAINTAINER: "cowan@bnl.gov"
+#    IM_NUMBER: ""
+    TESTS_GROUP: "pip"
+    SITE_ID: "nersc-$SITE"
+  script:
+    - source ${PIP_WORK_DIR}/${VENV_ENV_NAME}/bin/activate
+    - python3 ci/tests/test.py -s
+    - for TEST in flux parsl rp parsl-flux rp-flux maestro smartsim; do
```

Comment:
for pip-pipeline we keep these tests: `parsl rp maestro smartsim`
(python based packages, LLNL is exception regarding `flux`)

Reply:
pushed commit to remove flux tests